tests/core_tests.py

```diff
@@ -519,6 +519,15 @@ def test_templated_sql_json(self):
         data = self.run_sql(sql, "admin", "fdaklj3ws")
         self.assertEqual(data['data'][0]['test'], "2017-01-01T00:00:00")
 
+    def test_table_metadata(self):
+        dbid = self.get_main_database(db.session).id
+        data = self.get_json_resp(
+            "/caravel/table/{}/ab_user/null/".format(dbid))
+        self.assertEqual(data['name'], 'ab_user')
+        assert len(data['columns']) > 5
```

Comment:
could you please extend this test and check in case of MySQL db engine that the code returns: index, pk and fk ?